src/current/_includes/releases/v23.1/v23.1.13.md

```diff
@@ -0,0 +1,87 @@
+## v23.1.13
+
+Release Date: December 4, 2023
+
+{{ site.data.alerts.callout_info }}
+<release_version> is currently available only on select CockroachDB {{ site.data.products.dedicated }} and {{ site.data.products.serverless }} clusters. If you wish to upgrade your cluster to this version, please submit a [support request](https://support.cockroachlabs.com/hc/en-us/requests/new).
+
+This release is currently not available for CockroachDB {{ site.data.products.core }} customers.
+{{ site.data.alerts.end }}
+
+{% comment %}{% include releases/release-downloads-docker-image.md release=include.release %}{% endcomment %}
+
+<h3 id="v23-1-13-sql-language-changes">SQL language changes</h3>
+
+- A new option for the [`RESTORE`](https://www.cockroachlabs.com/docs/v23.1/restore.html) syntax, `strip_localities`, has been added. This can be used to strip the [locality information](https://www.cockroachlabs.com/docs/v23.1/alter-table.html#set-locality) from a backup when there are mismatched [cluster regions](https://www.cockroachlabs.com/docs/v23.1/multiregion-overview.html#cluster-regions) between the backup's cluster and the target cluster. The following are behaviors that will most likely not be encountered with the specific use case that this patch provides, but are documented nonetheless:      - Adding a [primary region](https://www.cockroachlabs.com/docs/v23.1/alter-database.html#set-primary-region) to a regionless restore (with or without [regional by row table(s)](https://www.cockroachlabs.com/docs/v23.1/table-localities.html#regional-by-row-tables)) will not work out-of-the-box, but does produce an accurate message detailing a user to essentially [`DROP TYPE [database].public.crdb_internal_region;`](https://www.cockroachlabs.com/docs/v23.1/drop-type.html) (and for cluster restores, [`ALTER DATABASE [database] CONFIGURE ZONE DISCARD;`](https://www.cockroachlabs.com/docs/v23.1/alter-database.html#remove-a-replication-zone)).
```

Comment:
ah sorry, they're absolute because we've had several changes to our link syntax in the last few months and particularly because of the way the release note includes are buried in folders I was having a hard time getting relative links to work.  There is also the pretty verbose Liquid full link syntax we're supposed to use elsewhere I think but I went with this in the release notes since the Liquid syntax is just as long but even more complicated IMO 🤷